test/e2e/integration/payment-test/payment.ts

```diff
@@ -0,0 +1,105 @@
+import 'cypress-iframe';
+
+import { faker } from '@faker-js/faker';
+import path from 'path';
+
+import { AppFrontend } from 'test/e2e/pageobjects/app-frontend';
+const appFrontend = new AppFrontend();
+
+let paymentUrl: string;
+
+describe('Payment', () => {
+  beforeEach(() => {
+    cy.startAppInstance(appFrontend.apps.paymentTest, { authenticationLevel: '2' });
+  });
+
+  it('Should fill out the form, landing on the payment page', () => {
+    cy.intercept({
+      method: 'GET',
+      url: '**/ttd/payment-test/*',
+      query: {
+        paymentid: '*',
+      },
```

Comment:
Doesn't this just catch all requests for the entire app? I'd assume that using a url like `**/payment` is preferable and less flaky.

Reply:
Or maybe this request isn't for payment at all? Are we requesting the app itself and expecting a redirect? :thinking: I'm not sure I'm 'getting' this, but I would think it could be more specific.

Reply:
We need this intercept because cypress has a hard time following the redirect back to our app. I've added a comment to explain what happens.